src/bes/nauru/lims/locales/senaite.patient.pot

```diff
@@ -16,74 +16,74 @@ msgstr ""
 msgid "If selected, users will be able to publish samples that have a Patient assigned with a temporary Medical Record Number (MRN)."
-msgstr "If selected, users will be able to publish samples that have a Patient assigned with a temporary Hospital Number."
+msgstr "If selected, users will be able to publish samples that have a Patient assigned with a temporary Medical Record Number (MRN)."
 
 #: senaite/patient/browser/controlpanel.py:211
 msgid "If selected, users will be able to verify samples that have a Patient assigned with a temporary Medical Record Number (MRN)."
-msgstr "If selected, users will be able to verify samples that have a Patient assigned with a temporary Hospital Number."
+msgstr "If selected, users will be able to verify samples that have a Patient assigned with a temporary Medical Record Number (MRN)."
 
 #: senaite/patient/adapters/listing.py:59
 #: senaite/patient/content/analysisrequest.py:92
 msgid "MRN"
-msgstr "Hospital Number"
+msgstr "MRN"
 
 #: senaite/patient/browser/controlpanel.py:201
 msgid "Make the MRN field mandatory in the sample registration form and when creating or modifying patients."
-msgstr "Make the Hospital Number field mandatory in the sample registration form and when creating or modifying patients."
+msgstr "Make the Medical Record Number (MRN) field mandatory in the sample registration form and when creating or modifying patients."
 
 #: senaite/patient/browser/patientfolder.py:75
 #: senaite/patient/content/analysisrequest.py:66
 msgid "Medical Record #"
-msgstr "Hospital Number"
+msgstr "Medical Record #"
 
 #: senaite/patient/adapters/listing.py:131
 msgid "Patient MRN of sample is not equal to %s"
-msgstr "Patient Hospital Number of sample is not equal to %s"
+msgstr "Patient Medical Record Number (MRN) of sample is not equal to %s"
 
 #: senaite/patient/content/patient.py:328
 msgid "Patient Medical Record # must be unique"
-msgstr "Patient Hospital Number must be unique"
+msgstr "Patient Medical Record Number (MRN) must be unique"
 
 #: senaite/patient/content/patient.py:109
 msgid "Patient Medical Record Number"
-msgstr "Patient Hospital Number"
+msgstr "Patient Medical Record Number (MRN)"
 
 #: senaite/patient/content/patient.py:322
 msgid "Patient Medical Record is missing or empty"
-msgstr "Patient Hospital Number is missing or empty"
+msgstr "Patient Medical Record Number (MRN) is missing or empty"
 
 #: senaite/patient/browser/controlpanel.py:200
 msgid "Require Medical Record Number (MRN)"
-msgstr "Require Hospital Number"
+msgstr "Require Medical Record Number (MRN)"
 
 #: senaite/patient/adapters/listing.py:39
 msgid "Temporary MRN"
-msgstr "Temp Hospital Number"
+msgstr "Temp MRN"
 
 #: senaite/patient/browser/viewlets/templates/temporary_mrn_viewlet.pt:13
 msgid "The Medical Record Number (MRN) assigned to this sample is temporary"
-msgstr "The Hospital Number assigned to this sample is temporary"
+msgstr "The Medical Record Number (MRN) assigned to this sample is temporary"
```

Comment:
Remove this entry, cause the `msgstr` equals to the original string defined in `senaite.patient`: https://github.com/senaite/senaite.patient/blob/a6e91b6601285255b3464e55c1e1808d25a37210/src/senaite/patient/locales/senaite.patient.pot#L406